src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,25 @@
+/** @jsx jsx */
+import { jsx, css } from "@emotion/react";
+import colors from "../colors";
+
+interface ErrorMessageProps {
+    show: boolean,
+    message: string
+}
+
+const styles = css`
+  color: ${colors.error};
+  font-size: 18px;
+  line-height: 15px;
+  margin: 15px 0 0;
```

Comment:
These should use [relative units](https://www.w3schools.com/cssref/css_units.asp). See `commonStyles.tsx` for a rough idea of how px map to rems, etc.